### DISCUSSION TITLE
MWPMYMA1 cert

### DISCUSSION THREAD

**Discussion by remylt** [flobz/psa_car_controller#421] [Q&A]
Hi, 

1 year ago we've the client cert MWPMYMA1.pfx to convert to .pem format for mitm.

I've re-download the lastest version of MyPsa app and now there is a password in MWPMYMA1.pfx. 

How can we do now ? because :
`openssl pkcs12 -in MWPMYMA1.pfx -out MWPMYMA1.pem -nodes`

don't work anymore

no one like me ?

**Comment by flobz** [comment]
Look at the app_decoder.py, the password is in it.

**Comment by remylt** [comment]
Thanks you for reply ! it was in _apk-parser.py_
how did you get this string ? plz
